structuredheadings/plugin.js

```diff
@@ -1,10 +1,11 @@
-/*eslint max-statements: [2, 25]*/ //I'll need to refactor somehow to meet this rule
+/*eslint max-statements: [2, 50]*/ //I'll need to refactor somehow to meet this rule
 (function () {
   CKEDITOR.plugins.add("structuredheadings", {
     icons: "autonumberheading," +
            "matchheading," +
            "increaseheadinglevel," +
-           "decreaseheadinglevel",
+           "decreaseheadinglevel," +
+           "restartNumbering",
```

Comment:
The only thing, is that simplifies the icons but I still need to list them all out individually somewhere, and that commandNames list would only be used for the icons anyway. This way is kinda cutting out the middleman.